### ISSUE TITLE
example folder needs cleanup

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/oopt-gnpy#109]
checked example folder and wondered about a few details:
- we have EDFA and EDFA model which seem to include similar but nt the same files. both are 5month old. do we really need both? which one is the correct one?

- taking edfa_model DFG_96.txt as an example: when clicking on it, it is shown as executable. This is strange for a text file and indeed it does not include code at all. Other .txt files are the same (I did not check all of them)

why do we need https://github.com/Telecominfraproject/oopt-gnpy/blob/develop/examples/Transmission%20Main%20Example.ipynb is anybody using it?

https://github.com/Telecominfraproject/oopt-gnpy/blob/develop/examples/std_medium_gain_advanced_config.json is located in examples but I wonder if that's the right place. Amplifiers are typically composed of EDFAs and those are in other folders. It's confusing to find it here.

**Comment by jeanluc-auge** [comment]
Hi Gert,
the /edfa is the original edfa model implementation example proposed before we had the transmission_main to test it. It is true we could remove it now.
In edfa_model, it is true we could do some cleaning but not too much so that it still shows how to build the edfa_config.json. 
Finally, the std_medium_gain_advanced_config.json is just an example of the advanced model input parameters for one of the amps defined in the eqpt library (eqpt_config.json). You can check, it is refered in. Therefore I think it has its place in the examples directory: it is not a generic file.

**Comment by ggrammel** [comment]
Jean-Luc,

thank you for looking at it.
see below ...


Sent from my iPhone

On 20. Sep 2018, at 16:57, Jean-Luc Augé <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi Gert,
the /edfa is the original edfa model implementation example proposed before we had the transmission_main to test it. It is true we could remove it now.

agree


In edfa_model, it is true we could do some cleaning but not too much so that it still shows how to build the edfa_config.json.

I believe we can largely leave as it is for now if we can remove the edfa directory.


Finally, the std_medium_gain_advanced_config.json is just an example of the advanced model input parameters for one of the amps defined in the eqpt library (eqpt_config.json). You can check, it is refered in. Therefore I think it has its place in the examples directory: it is not a generic file.

This was a general thought. My usecase was to try to figure out from the file structure where to put a model for another amplifier and which files would need to be touched.
I know we are not yet at this level of perfection, but think it‘s a useful exercise to determine what we should improve.

Gert

**Comment by ggrammel** [comment]
Jonas' comments triggered me to have a closer look at our files and after a while I kind of concluded that I would need help to understand. My aim was to read the documentation, then look at the files and try to understand what they are doing or which parameters are supposed to be described there. The starting point was supposed to be simple, so I looked at edfa/edfa_model

-	It is very confusing to see 2 directories “edfa” and “edfa_model”. Initially I tought “edfa_model” is the one used, but in Esther’s Excel user guide it says “edfa”.
-	In edfa:
o	Pchan2D.txt: not much text here, so presumably I need to look at Pchan2DLegend.txt for a clue
o	Pchan2DLegend.txt: again just numbers, no description. Why is this called “Legend” in the first place?
o	Edfa_config.json: Looks like the description of an EDFA, but the numbers there do not match any number in the first 2 files, so those do not seem to be related
o	Example.py: finally some executable code but not many comments which seems to use the first files - but it doesn’t find 'edfa_example_network.json'
-	In edfa_model:
o	DFG_96, DGT_96, NFR_96: not much to understand there
o	OA.json references the files above – so far so good
o	Pchan2D.txt, Pchan2DLegend.txt: contain just numbers
o	Amplifier.py: finally code with comments to learn something, but the code doesn’t run. It looks to import from “utilities” which doesn’t seem to exist
o	Build_oa.py: not 100% clear how it needs to be used. I tried but probably added the wrong parameters
o	Edfa_config.json: that looks like it would be an output created by Build_oa.py. At least that’s what is my understanding
o	pNFfit3.txt: not much to understand from the file

It would be great if there would be a simple way to add a comment line to each file with numbers to at least tell what it is and a hint which python module consumes it. 
If that is too difficult, a Readme could be generated to briefly describe what can be found where and put in the same directory
